python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1125,16 +1125,19 @@ def convert_dropout(node, **kwargs):
     opset_version = kwargs["opset_version"]
 
     probability = float(attrs.get("p", 0.5))
+    axes = convert_string_to_list((attrs.get("axes", "None")))
+    mode = attrs.get('mode', 'training')
 
-    if opset_version >= 12:
-        # opset >= 12 requires the ratio to be an input
-        nodes = [
-            create_const_scalar_node(name+"_ratio0", np.float32(probability), kwargs),
-            make_node("Dropout", [input_nodes[0], name+"_ratio0"], [name], name=name)
-        ]
-        return nodes
-    else:
-        return [make_node("Dropout", input_nodes, [name], ratio=probability, name=name)]
+    if axes != [None]:
```

Comment:
Do you need to check for string value 'None'?

Reply:
convert_string_to_list() will return [None] if input is 'None'